pkg/apis/core/types.go

```diff
@@ -1680,6 +1680,15 @@ type CSIPersistentVolumeSource struct {
 	// secret object contains more than one secret, all secrets are passed.
 	// +optional
 	ControllerExpandSecretRef *SecretReference
+
+	// NodeExpandSecretRef is a reference to the secret object containing
+	// sensitive information to pass to the CSI driver to complete the CSI
+	// NodeExpandVolume call.
+	// This is an alpha field and requires enabling ExpandCSIVolumes feature gate.
```

Comment:
`ExpandCSIVolumes` is beta, not alpha.

Reply:
done, both modify the ControllerExpandSecretRef